src/Engine/UnitMods/UnitModifier.java

```diff
@@ -53,4 +55,7 @@ public interface UnitModifier extends Serializable
   void modifyCargoCapacity(UnitContext uc);
   void modifyActionList(UnitContext uc);
 
+  /** Convenience hook for StateTracker injection */
+  default void registerTrackers(GameInstance gi) {}
```

Comment:
Should this do anything? It doesn't look like any subclasses populate the method either.

Reply:
It's used in the next PR.